perfkitbenchmarker/pkb.py

```diff
@@ -506,12 +503,18 @@ def RunBenchmarks(publish=True):
   for i, benchmark_tuple in enumerate(benchmark_tuple_list):
     benchmark_module, user_config = benchmark_tuple
     benchmark_name = benchmark_module.BENCHMARK_NAME
+    benchmark_config_dict = benchmark_module.GetConfig(user_config)
```

Comment:
That particular change would be creating a function for two statements (five lines of code) that's used only here. I do see that RunBenchmarks is getting a bit large, but I think a separate PR could break it up a bit better.